onnxruntime/contrib_ops/cpu/matmul_integer.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/common/common.h"
+#include "core/framework/op_kernel.h"
+#include "core/util/math_cpuonly.h"
+
+namespace onnxruntime {
+namespace contrib {
+
+template <typename T1, typename T2, typename T3>
+class MatMulInteger final : public OpKernel {
+ public:
+  MatMulInteger(const OpKernelInfo& info) : OpKernel(info) {
+    if (info.GetInputCount() > 2) {
```

Comment:
if the input count is 3, then a has zero point; if the input count is 4 then b has zero point too. make this logic explicit please.